vmaas_sync/vmaas_sync.go

```diff
@@ -215,6 +218,10 @@ func RunVmaasSync() {
 
 	go RunSystemCulling()
 
+	if enableRefreshPackagesCache {
```

Comment:
please, use the bool var inside the function `refreshLatestPackagesCount`, as done for other methods.

Reply:
Fixed